tests/src/Mod/Part/App/TopoShapeMakeShapeWithElementMap.cpp

```diff
@@ -136,31 +131,10 @@ TEST_F(TopoShapeMakeShapeWithElementMapTests, mapCompoundCount)
     EXPECT_EQ(postElements.count(IndexedName("Vertex", 17)), 0);
     EXPECT_EQ(postElements.count(IndexedName("Face", 12)), 1);
     EXPECT_EQ(postElements.count(IndexedName("Face", 13)), 0);
-    EXPECT_STREQ(sources[0].shapeName().c_str(), "Solid");
-    EXPECT_STREQ(sources[1].shapeName().c_str(), "Solid");
+    EXPECT_STREQ(sources[0].shapeName().c_str(), "Compound");
+    EXPECT_STREQ(sources[1].shapeName().c_str(), "Compound");
     EXPECT_STREQ(compound.shapeName().c_str(), "Compound");
-}
-
-TEST_F(TopoShapeMakeShapeWithElementMapTests, mapCompoundMap)
-{
-    // Arrange
-    auto [cube1, cube2] = PartTestHelpers::CreateTwoCubes();
-    std::vector<TopoShape> sources {cube1, cube2};
-    sources[0].Tag = 1;
-    sources[1].Tag = 2;
-    // Map only one of the two sources to test different names.
-    sources[0].makeShapeWithElementMap(sources[0].getShape(), *Mapper(), {sources[0]});
-    TopoShape compound = TopoShape();
-    compound.makeElementCompound(sources);
-    auto preElements = elementMap(compound);  // Map before mapping.
-    // Act
-    compound.makeShapeWithElementMap(compound.getShape(), *Mapper(), sources);
-    auto postElements = elementMap(compound);  // Map after mapping
-    // Assert
-    EXPECT_EQ(preElements[IndexedName("Edge", 1)], MappedName("Edge1;MAK;:H:4,E;:H1:b,E"));
-    EXPECT_EQ(preElements[IndexedName("Edge", 13)], MappedName("Edge1;:H2,E"));
-    EXPECT_EQ(postElements[IndexedName("Edge", 1)], MappedName("Edge1;MAK;:H:4,E;MAK;:H1:f,E"));
-    EXPECT_EQ(postElements[IndexedName("Edge", 13)], MappedName("Edge1;MAK;:H2:4,E"));
```

Comment:
Why is this test removed?

Reply:
It is redundant with other tests.  The pre / post elements no longer applies with the = operator working correctly, because the element map is there and the same.